handlers.js

```diff
@@ -159,6 +309,31 @@
 
             // strip '/' off end of path
 
+            if (this.app.opts.optExcludeDotHtml && ! this.request.origpath.endsWith("/")) {
+                var extension = this.request.path.split('.').pop();
+                var more = this.request.uri.split('.html').pop()
+                if (extension == 'html') {
```

Comment:
people used to also write `.htm` files though I guess that's back from a long time ago :-)

Reply:
Yeah I did think about that. If someone makes an issue I may implement it but that would make the whole thing just crazy